tools/installbazel/main.sh

```diff
@@ -18,7 +18,7 @@
 
 set -e
 
-function install_bazel() {
+function install_bazel_x86_64() {
```

Comment:
Can we use bazelisk for every arch?

Reply:
I think we can use bazelisk for x86_64 as well. You could see `bazelisk-linux-amd64` [here](https://github.com/bazelbuild/bazelisk/releases).


Should the version of `bazel` sync with any other places beyond this github repo? \
Unless, using `bazelisk` for `arm64` only would diverge the version used for `bazel`.

But I prefer to do this in the following PR, not this one. Since the primary goal is fixing build issue for ARM64 first.